frontend/src/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -9,11 +37,11 @@ class CrossingDetails extends Component {
       <div>
         <div>Crossing Details: {crossing.id}</div>
         <div> {crossing.humanCoordinates} </div>
-        <input type='text' defaultValue={crossing.name}></input>
+        <input type="text" value={this.state.name} onChange={this.nameChanged}/>
         <div> {crossing.humanAddress} </div>
-        <input type='text' defaultValue={crossing.description}></input>
-        <input type='text' defaultValue={community.name}></input>
-        <div> SAVE CROSSING </div>
+        <input type="text" value={this.state.description} onChange={this.descriptionChanged}/>
+        <div>{community.name}</div>
+        <button onClick={this.updateCrossing.bind(this)}> SAVE CROSSING </button>
```

Comment:
Probably want to bind this to the handler as you did in CrossingListHeader.js here: https://github.com/cityofaustin/ctxfloods/blob/master/frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.js#L24